src/test/java/uk/gov/hmcts/reform/coh/controller/utils/CommonMessagesTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.coh.controller.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CommonMessagesTest {
+
+    @Test
+    public void testInitateCommonMessages() {
+        CommonMessages cm = new CommonMessages();
```

Comment:
Is this test really necessary?

Reply:
It doesn't get picked up by SonarQube otherwise.

Reply:
I think that class would've been better implemented as an Enum. For now, add assertTrue after the constructor to prove it got past that point. Otherwise the test isn't actually asserting a condition.

Reply:
Or .... just make it a private constructor